src/bayesdb.py

```diff
@@ -128,6 +142,49 @@ def sql_untrace(self, tracer):
         assert self.sql_tracer == tracer
         self.sql_tracer = None
 
+    def check_uncompleted_session_entries(self):
+        '''Check if the previous session ended with a failed command and
+        suggest sending the session'''
+        cursor = self.sqlite3.execute('''SELECT COUNT(*) FROM
+        bayesdb_session_entries WHERE completed=0 AND session_id=?''', (self.session_id-1,))
+        uncompleted_entries = int([row[0] for row in cursor][0])
+        if uncompleted_entries > 0:
+            print '''WARNING: Previous session contains uncompleted entries. This may be due to a bad termination or crash of the previous session. Consider uploading the session.'''
```

Comment:
Not sure what the right warning mechanism is, but printing to stdout is unlikely to be it. Let's ask @riastradh-probcomp ?